frontend/src/views/ProjectError.vue

```diff
@@ -0,0 +1,69 @@
+<!--
+SPDX-FileCopyrightText: 2020 SAP SE or an SAP affiliate company and Gardener contributors
+
+SPDX-License-Identifier: Apache-2.0
+ -->
+<template>
+  <g-error
+    :code="code"
+    :text="text"
+    :message="message"
+    :buttonText="buttonText"
+    @click="onClick"
+  ></g-error>
+</template>
+
+<script>
+import GError from '@/components/GError'
+import get from 'lodash/get'
+
+export default {
+  components: {
+    GError
+  },
+  props: {
+    code: {
+      type: [String, Number],
+      default: '404'
+    },
+    text: {
+      type: String,
+      default: 'Project not found'
+    },
+    message: {
+      type: String,
+      default: 'The project you are looking for doesn\'t exist or an other error occured!'
+    },
+    buttonText: {
+      type: String,
+      default: 'Get me out of here'
+    }
+  },
+  computed: {
+    fallbackRoute () {
+      const defaultNamespace = this.$store.getters.defaultNamespace
+      const namespace = get(this.$route, 'params.namespace', defaultNamespace)
+      if (namespace) {
+        return {
+          name: 'ShootList',
+          params: {
+            namespace
+          }
+        }
+      }
+      return {
+        name: 'Home'
+      }
+    }
+  },
```

Comment:
this does not work anymore:
- login e.g. with service account token that is not a member of any project
- navigate to `/namespace/garden-notexisting/shoots/notexisting`
- Clicking the `Get me out of here` button will not bring you to the `Home` screen

Reply:
✔️